.github/workflows/R-CMD-check_docker.yaml

```diff
@@ -8,17 +8,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        r_version: ["4.1.3", "4.2.3", "4.3.3", "4.4.2"]
+        r_version: ["4.1.3", "4.2.3", "4.3.3", "4.4.3"]
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       -
         name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
       -
         name: Build docker image
-        uses: docker/build-push-action@v4
+        uses: docker/build-push-action@v5
```

Comment:
newest version of build-push-action is version 6. Is there a reason to use version 5 over version 6? (https://github.com/docker/build-push-action/releases)

Reply:
Good point. Will fix.